### PR TITLE
[13.0][ADD] edi_voxel_secondary_unit: new module

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT28726

This module extends the functionality of 'Product Secondary Unit' module to support Voxel Units of Measure codes in 'Secondary Units of measure'.

This is a base module for the following modules:

* edi_voxel_sale_secondary_unit
* edi_voxel_stock_picking_secondary_unit

### CODE REVIEW

File: edi_voxel/models/uom_uom.py [hunk 1]
```diff
@@ -3,23 +3,23 @@
 
 from odoo import fields, models
 
+VOXEL_CODE = [
+    ("Unidades", "Unidades"),
```
Comment (pedrobaeza) [inline edi_voxel/models/uom_uom.py]: Take the occasion to use the string to English and translate it on es.po.

File: edi_voxel_secondary_unit/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "summary": "Add Voxel UoM code to Secondary UoM model",
+    "version": "13.0.1.0.0",
+    "development_status": "Production/Stable",
+    "category": "Hidden",
+    "website": "https://github.com/OCA/edi",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "maintainers": ["ernestotejeda"],
+    "license": "AGPL-3",
+    "depends": ["product_secondary_unit", "edi_voxel"],
+    "data": ["views/product_secondary_unit_views.xml"],
```
Comment (pedrobaeza) [inline edi_voxel_secondary_unit/__manifest__.py:14]:
```suggestion
    "data": ["views/product_secondary_unit_views.xml"],
    "installable": True,
    "auto_install": True,
```

**Comment by ernestotejeda** [comment]
@pedrobaeza changes done

**Comment by pedrobaeza** [comment]
You need to change the translations as well on original `edi_voxel` module.

**Comment by ernestotejeda** [comment]
@pedrobaeza done

**Review by pedrobaeza — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**